CustomCommandExamples/README.md

```diff
@@ -88,6 +89,14 @@ See [Installing DataMiner Integration Studio](https://aka.dataminer.services/Dis
 
 2. The custom command will be executed and the JSON example should be displayed as the response. <details><summary>`show demo`</summary>![Gif-OutputJson](https://user-images.githubusercontent.com/33500507/192817316-fc4de34d-a34e-42cd-a5fa-e8e99b363c2d.gif)</details>
 
+#### Input Parameter
+
+1. In Teams, execute the command '*run Input Parameter Example*'.
+
+2. A prompt will request the input configured in the automation script. Fill in any value and click the '*Run*' button.
```

Comment:
The [Microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/prompt) recommends not using the noun "prompt" for anything other than the command prompt:

```suggestion
2. When you are asked to provide the input configured in the Automation script, fill in any value and click the '*Run*' button.
```